### DISCUSSION TITLE
letsencrypt 405 Method Not Allowed when attempting a certificate renewal

### DISCUSSION THREAD

**Discussion by btakita** [nextcloud/all-in-one#3504] [Questions] [answered]
### Steps to reproduce
1. Letsencrypt cert expires
2. Restart nextcloud-aio-apache

### Expected behavior 
The letsencrypt cert renewal should be successful.

### Actual behavior 
405 Method Not Allowed error from the letsencrypt service.

### Host OS 
Arch Linux

#### Nextcloud AIO version 
v7.2.1

#### Current channel 
latest

#### Other valuable info 
See https://github.com/nextcloud/all-in-one/discussions/3497

After running `php occ app:update --all`, the same error occurs.

**Comment by szaimen** [comment, marked as answer]
Duplicate of https://github.com/nextcloud/all-in-one/discussions/3497
